cmd/src/campaigns_suggestions.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"context"
+
+	"github.com/sourcegraph/src-cli/internal/api"
+	"github.com/sourcegraph/src-cli/internal/campaigns"
+	"github.com/sourcegraph/src-cli/internal/output"
+	"gopkg.in/yaml.v2"
+)
+
+// prettyPrintCampaignsUnlicensedError introspects the given error returned when
+// creating a campaign spec and ascertains whether it's a licensing error. If it
+// is, then a better message is output. Regardless, the return value of this
+// function should be used to replace the original error passed in to ensure
+// that the displayed output is sensible.
+func prettyPrintCampaignsUnlicensedError(ctx context.Context, out *output.Output, svc *campaigns.Service, spec *campaigns.CampaignSpec, err error) error {
+	// Pull apart the error to see if it's a licensing error: if so, we should
+	// display a friendlier and more actionable message than the usual GraphQL
+	// error output.
+	if gerrs, ok := err.(api.GraphQlErrors); ok {
+		// A licensing error should be the sole error returned, so we'll only
+		// pretty print if there's one error.
+		if len(gerrs) == 1 {
+			if code, cerr := gerrs[0].Code(); cerr != nil {
+				// We got a malformed value in the error extensions; at this
+				// point, there's not much sensible we can do. Let's log this in
+				// verbose mode, but let the original error bubble up rather
+				// than this one.
+				out.Verbosef("Unexpected error parsing the GraphQL error: %v", cerr)
+			} else if code == "ErrCampaignsUnlicensed" {
+				return outputUnlicensedError(ctx, out, svc, spec)
+			}
+		}
+	}
+
+	// In all other cases, we'll just return the original error.
+	return err
+}
+
+func outputUnlicensedError(ctx context.Context, out *output.Output, svc *campaigns.Service, spec *campaigns.CampaignSpec) error {
+	// OK, let's print a better message, then return an exitCodeError to
+	// suppress the normal automatic error block.
+	//
+	// First, we're going to print a generic message about the restrictions on
+	// deployments without a campaigns license.
+	block := out.Block(output.Line("🪙", output.StyleWarning, "Campaigns is a paid feature of Sourcegraph. All users can create sample campaigns with up to five changesets without a license."))
```

Comment:
```suggestion
	block := out.Block(output.Line("🪙", output.StyleWarning, "Campaigns is a paid feature of Sourcegraph. All users can create sample campaigns with up to 5 changesets without a license."))
```

Usually I wouldn't recommend this (I remember a rule saying that every number up to 12 should be written out), but in this case I think the number should stand out.

Reply:
Implemented in the rebase.